<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>АСУКТ ВЦ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление температурой</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Минимальная температура(C):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>15.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальная температура(C):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление влажностью</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Минимальная влажность(%):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальная влажность(%):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>80.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>120</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Кабинеты</string>
    </property>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="value">
      <number>19</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="value">
      <double>60.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="value">
      <double>19.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Температура:</string>
     </property>
    </widget>
    <widget class="QDial" name="dial_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Влажность:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>561</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Отчет о состоянии системы</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>681</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Лог ошибок и предупреждений:</string>
    </property>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>671</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>681</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Состояние системы</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>85</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-Kirillov-Desktop_Qt_5_3_GCC_64bit-Debug/sysoff.png</normaloff>../build-Kirillov-Desktop_Qt_5_3_GCC_64bit-Debug/sysoff.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="allBySelf"/>
    <addaction name="allAuto"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menuSds">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="authors"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSds"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="allBySelf">
   <property name="text">
    <string>Ручной ввод</string>
   </property>
  </action>
  <action name="allAuto">
   <property name="text">
    <string>Автоматический ввод</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Данный пункт меню позволяет покинуть сиё творение человеческой мысли, реализованное посредством множества кмопьютерных операций.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="authors">
   <property name="text">
    <string>Авторы</string>
   </property>
   <property name="toolTip">
    <string>Кириллов пупсик</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>doubleSpinBox_4</tabstop>
  <tabstop>listWidget_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
